python/ray/autoscaler/updater.py

```diff
@@ -118,15 +112,21 @@ def set_ssh_ip_if_required(self):
         #   the ControlPath directory exists, allowing SSH to maintain
         #   persistent sessions later on.
         with open("/dev/null", "w") as redirect:
-            self.get_caller(False)(
-                ["mkdir", "-p", self.ssh_control_path],
-                stdout=redirect,
-                stderr=redirect)
+            try:
```

Comment:
Should these exit of fail?

Reply:
Hm, well they didn't exit on fail before so I tried to keep backwards compat